cli/tests/test_observability_v8_redaction_policy.py

```
@@ -0,0 +1,255 @@
+    )
+    assert _destination(edited_source, "archive")["routes"][1] == edited
+    _, removed = _apply(
+        edited_bytes,
+        route_remove_mutations(edited_source, "archive", "finding"),
+    )
+    assert [route["name"] for route in _destination(removed, "archive")["routes"]] == ["model"]
+
+
+def test_generated_destinations_are_read_only() -> None:
+    source = _apply(_source(), ())[1]
+    with pytest.raises(ValueError, match="generated destination.*read-only"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)